procedures/LVV-T2572 - Rotator Duty Cycle Tests.ipynb

```diff
@@ -0,0 +1,822 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>You forgot to compute the ra/dec from az/el here... but this actually needs to be done on the previous cell, when you are slewing to a new target while keeping the rotator at zero degrees.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Fixed this.</p>